STYLE.md

```diff
@@ -0,0 +1,31 @@
+# Style Guide
+
+To ensure the consistency of a comprehensive documentation like grammY’s, there must also be a set of rules for humans to follow.
+This documents those rules.
+
+## 0
+
+grammY’s name must always be written as “grammY”, regardless of its position in the sentence, the casing of its surroundings, and the language of its context.
+
+```diff
+- Grammy---the Telegram Bot Framework
++ grammY---the Telegram Bot Framework
+```
+
+## 2
+
+The first letter of the names of grammY entities (e.g., plugin names, chat names) must be in lowercase, regardless of the casing of its surroundings.
+
+```diff
+- grammY Runner
++ grammY runner
+```
+
+## 3
+
+In English, the the headings must be in title case and follow the rules of [AP Stylebook](https://en.wikipedia.org/wiki/Title_case#AP_Stylebook).
+
+```diff
+- ## Notes regarding the possibilities of facing errors
++ ## Notes Regarding the Possibilities of Facing Errors
```

Comment:
```suggestion
- ## with great style comes great responsibility
+ ## With Great Style Comes Great Responsibility
```
is funnier but a worse example, you decide

Reply:
Nice!